### ISSUE TITLE
cli failures

### ISSUE THREAD

**Issue by ysard** [labsquare/cutevariant#173]
TL;DR: Nothing works, and I will end up in a nervous breakdown :rage: 

- [x] --db argument is not functional
- [x] exec statements are not functional

> sql.QueryBuilder(conn)
> AttributeError: module 'cutevariant.core.sql' has no attribute 'QueryBuilder'

- [x] What is the form of the data in group arg of exec statement ? Please put it in the example designed for this.

- [x] createdb statement is a failure:

> File "/media/DATA/Projets/cutevariant/cutevariant/cutevariant/core/cli.py", line 117, in main
> async_import_file(conn, args.input), redirect_stdout=True
> TypeError: 'module' object is not callable

In setup.cfg i see:
> progressbar>=2.5

In the code i see:

    import progressbar
    for i in progressbar.progressbar(async....):

Have you confused the dependency with the progressbar2 package ?

https://pypi.org/project/progressbar/
vs
https://pypi.org/project/progressbar2/

The progressbar2 package has this code example in its doc:

> import progressbar
> for i in progressbar.progressbar(range(100)):

The packages are imported with the same name which is a F**K**G bad practice from the devs, BUT in Cutevariant side, this code was not tested before being pushed!

**Comment by ysard** [comment]
EDIT: ok imports are fixed; 

- [x] I let you fix exec statement.

- [ ] also, the progressbar displays the number of lines displayed: This is quite useless can you fix that ?

> - |                      #                                                                | 1141 Elapsed Time: 0:21:25

=> 1141 lines...

**Comment by ysard** [comment]
VQL exceptions handling is crap:
```
except vql.textx.TextXSyntaxError as e:
            print(
                "==================================== ERRORS ===================================="
            )
            print("TextXSyntaxError: %s, col: %d" % (e.message, e.col))
            print(" ")
            print(query)
            print("_" * (e.col - 1) + "^\n")
            exit(0)
```

- [x] Can you implement magic mathod `__str__` or `__repr__` in such to avoid this bash/R/Perl/JS like lines ?
 (See VQLSyntaxError in ./cutevariant/core/vql.py).
Moreover exceptions should be logged with... a logger. Thus, using `__repr__` will allow to just use `LOGGER.critical(my_exception)`
and also have a nice output.

**Comment by ysard** [comment]
Done, except the correct % in progressbar. Can't seem to be better in our situation.
